src/home-page/data/data.ts

```diff
@@ -0,0 +1,50 @@
+import backendIcon from '../../icons/backend.svg';
+import devopsIcon from '../../icons/devops.svg';
+import frontendIcon from '../../icons/frontend.svg';
+import projectIcon from '../../icons/project_manager.svg';
+import qaIcon from '../../icons/qa.svg';
+import uxIcon from '../../icons/ux_ui.svg';
```

Comment:
Mayby we can use one file with all svg icons together? To limit imports. What do you think? This solution is also fine of course, and we can stay with that :)

Reply:
@OlgaMilczek to be honest I had no idea that you can have multiple svg in one file and use them separetly, so a good lesson for me today :) (but I'm not sure, if it was easy/possible to implement in this particular case). Anyway, I would leave it as it is. It's clear and easy to read or update the fille in case you want to change one of the icons in the future.

Reply:
@klaraborowska Yes you are right I confused import svgs in html and js. :( My bad. @ireshka Sorry I is perfect :)

Reply:
We could make these `svgs` react components rather than just svg files and then we would have more options concerning imports / exports :) But this is just a note, I agree that the current solution is just fine :ok_hand: